docs/software/overview.md

```diff
@@ -0,0 +1,63 @@
+---
+tags:
+ - Software
+---
+# Software Overview
+
+Each ORCD system has its own software stack. Many basic and commonly used software and libraries are already installed, so it is good to check before spending the time to install it yourself. This page discusses the general overview of what kinds of software are supported and points you to how to use them and what to do if what you need isn't there.
+
+## Software Landscape
+
+While the software stack will be different on each system, there are three general classes of software:
+
+- Core: Commonly used or fundamental software and libraries that are fully supported. Core software is expected to work until it is officially deprecated, and often newer versions are provided to replace them.
+- Community: Software that has been built and installed by request, but is not commonly used. Support is on a best-effort basis. Should work when built but not guaranteed to work indefinitely or replaced with newer versions when deprecated, except by request.
+- Deprecated: Software that is no longer supported or expected to work. May be kept for legacy reasons, or will soon be removed.
```

Comment:
Maybe include resources for if the user's software is deprecated or going to be deprecated? If I was dealing with that software, I would want the next steps on where I could potentially request an updated version of the said software.

Reply:
Added suggestion to move to a newer version (if available), request a newer version (if not available), or try Singularity to run the older version